amical/tests/test_tools.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import pytest
+
+from amical import tools
+
+
+def test_crop_max():
+
+    img_size = 80  # Same size as NIRISS images
+    img = np.ones((img_size, img_size))
+    xmax, ymax = 17, 57
+    img[ymax, xmax] = img.max() * 5  # Add off-centered max pixel
+
+    # Pre-calculate expected max size
+    isz_max = 2 * np.min([xmax, img.shape[1] - xmax, ymax, img.shape[0] - ymax])
+    isz_too_big = isz_max + 1
+
+    # Using full message because we also check the suggested size
+    size_msg = (
+        f"The specified cropped image size, {isz_too_big}, is greater than the distance"
+        " to the PSF center in at least one dimension. The max size for this image is"
+        f" {isz_max}."
```

Comment:
You probably don't want to add a `.` right next to a number. Also your other errors/warnings don't end with a dot either
```suggestion
        f" {isz_max}"
```